packages/karma-puppeteer-launcher/package.json

```diff
@@ -30,6 +30,6 @@
   },
   "peerDependencies": {
     "karma": "*",
-    "puppeteer": "*"
```

Comment:
This can actually stay `*`. The version in the main `package.json` is what matters.